app/src/main/java/cto/its/ny/gov/fakenews/model/Article.kt

```diff
@@ -1,37 +1,10 @@
 package cto.its.ny.gov.fakenews.model
 
+import android.os.Parcel
+import android.os.Parcelable
 import com.google.gson.annotations.SerializedName
+import kotlinx.parcelize.Parcelize
 
-
-data class Article (
-
-    @SerializedName("source"      ) var source      : Source?,
-    @SerializedName("author"      ) var author      : String = "Annonymous",
-    @SerializedName("title"       ) var title       : String,
-    @SerializedName("description" ) var description : String? = null,
-    @SerializedName("url"         ) var url         : String,
-    @SerializedName("urlToImage"  ) var urlToImage  : String? = null,
-    @SerializedName("publishedAt" ) var publishedAt : String,
-    @SerializedName("content"     ) var content     : String? = null
-
-
-) {
-    override fun toString(): String {
-        var concatString: String = ""
-        var _title: StringBuilder = StringBuilder();
-        var _author = "";
-        var _srcName = ""
-        if (this.title != null) {
-            _title.append("  Title: ")
-            _title.append(this.title)
-            concatString = concatString.plus(_title.toString())
-        }
-        if (_author != null) {
-            concatString = concatString.plus("  Author: ${author}")
-        }
-        if (source?.name != null) {
-            concatString = concatString + "  Source: " + source?.name
-        }
-        return concatString
-    }
-}
+@Parcelize
+data class Article(val author: String? = null, var title: String , var content: String, var description: String? = null, var publishedAt: String , var source: Source , var url: String , var urlToImage: String? = null ) :
+    Parcelable

```

Comment:
Great work on using `@Parcelize`! 

Minor note, it's typically recommended to have each property on it's own line for readability:

```suggestion
@Parcelize
data class Article(
    val author: String? = null, 
    var title: String , 
    var content: String, 
    ...,
    var urlToImage: String? = null 
) : Parcelable
```